cmd/agent/daemon/state/controller.go

```diff
@@ -54,6 +53,7 @@ type ebpfTracer interface {
 	UnmuteEventsFromCgroups(cgroups []uint64) error
 	IsCgroupMuted(cgroup uint64) bool
 	ReadSyscallStats() (map[ebpftracer.SyscallStatsKeyCgroupID][]ebpftracer.SyscallStats, error)
+	HarvestNetworkSummary() (map[ebpftracer.TrafficKey]ebpftracer.TrafficSummary, error)
```

Comment:
Harvest sounds a bit unusual 😄 
Maybe `CollectNetworkSummary` or `ReadNetworkSummary` as we already have similar `ReadSyscallStats`

Reply:
Thank you! IDK but I couldn't think of a better name 😅 I will rename it to `CollectNetworkSummary` 😅